### PR TITLE
Week1 portfolio

### PR DESCRIPTION
**Pull request by BenMob**
Here is the live URL : [bombeni1-sps-summer20.uc.r.appspot.com/](url)

Please review when able!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Google Software Product Sprint
 
-This repo contains the projects you'll work on throughout SPS.
+This repo contains Benjamin's portfolio and SPS projects.
```
Comment (gitmnk) [inline README.md]: SPS 2020 projects

File: portfolio/src/main/webapp/css/home.css [hunk 2]
```diff
@@ -0,0 +1,86 @@
+#my-portrait-img{
```
Comment (gitmnk) [inline portfolio/src/main/webapp/css/home.css:1]: Alpahbetize declarations. This is a great practice for coding and makes it easy to refactor :)

File: portfolio/src/main/webapp/css/home.css [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+    font-size: 30px;
+    border-radius: 15%;
+    color: #fff;
+    background: #d6249f;
+    background: radial-gradient(circle at 30% 107%, #fdf497 0%, #fdf497 5%, #fd5949 45%,#d6249f 60%,#285AEB 90%);
+    box-shadow: 0px 3px 10px rgba(0,0,0,.25);
+}
+.instagram:hover{
+    box-shadow: 0px -10px 5px #5B5BE9;
+}
```
Comment (gitmnk) [inline portfolio/src/main/webapp/css/home.css]: add a new line to end of file.

File: portfolio/src/main/webapp/css/projects.css [hunk 4]
```diff
@@ -0,0 +1,10 @@
+#cc-demo:hover{
+    border-radius: 10px;
+    padding: 5px;
+    background-color: rgb(0, 36, 61)
+}
+#ss-demo:hover{
+    border-radius: 10px;
+    padding: 5px;
+    background-color: rgb(0, 36, 61)
+}
```
Comment (gitmnk) [inline portfolio/src/main/webapp/css/projects.css]: Add a new line to End of file

File: portfolio/src/main/webapp/css/style.css [hunk 5]
```diff
@@ -0,0 +1,199 @@
...
+}
+
+.content{
+    color:whitesmoke;
+    bottom: 0%;
+    width: 960px;
+    margin-left: auto;
+    margin-right:auto;
+    background-color: rgb(0, 36, 61);
+   /* background-color: rgb(43, 72, 107);*/
```
Comment (gitmnk) [inline portfolio/src/main/webapp/css/style.css]: Remove the commented line.

File: portfolio/src/main/webapp/css/style.css [hunk 6]
```diff
@@ -0,0 +1,199 @@
...
+    box-shadow: 0px -5px 30px black;
+}
+
+.navbar{
```
Comment (gitmnk) [inline portfolio/src/main/webapp/css/style.css]: the coding style guide seems to point out  a space before the starting {.
https://google.github.io/styleguide/htmlcssguide.html#CSS_Formatting_Rules

File: portfolio/src/main/webapp/css/style.css [hunk 7]
```diff
@@ -0,0 +1,199 @@
...
+}
+
+.padding20{
+   padding: 20px; 
+}
+.margin-bottom-100{
+   margin-bottom: 100px;
+}
+
+/* TODO: Rename this */
```
Comment (gitmnk) [inline portfolio/src/main/webapp/css/style.css]: Need renaming still ?

File: portfolio/src/main/webapp/index.html [hunk 8]
```diff
@@ -1,18 +1,75 @@
...
+                <div class="flexbox-container-even black-shadow" id="about">
+                    <div>
+                        <a  href="projects.html#cougar-roumie">
+                            <img class="yellow-shadow-onhover" id=my-portrait-img src="images/laptop.jpg" href="projects.html#cougar-roumie">
+                        </a>  
+                    </div>
+                    <div id="welcome-msg-wrapper">
+                        <h2 id="welcome-msg" ><em>Hi, I am Ben</em><h2> 
+                        <p id="intro-par">I am a rising third year Software Engineering Student at <a href="https://www.csusm.edu/">California State University San Marcos</a>. Now, how did I end up in this field ?
+                            During my final year of High School, I was indecisve about what I wanted to do. So I decided to take an Introduction to Computer Science course
```
Comment (gitmnk) [inline portfolio/src/main/webapp/index.html]: typo in indecisve

File: portfolio/src/main/webapp/js/script.js [hunk 9]
```diff
@@ -26,3 +26,17 @@ function addRandomGreeting() {
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
 }
+
```
Comment (gitmnk) [inline portfolio/src/main/webapp/js/script.js:29]: remove empty line

File: portfolio/src/main/webapp/js/script.js [hunk 10]
```diff
@@ -26,3 +26,17 @@ function addRandomGreeting() {
...
+
+/***
+    Highlight the current section on the navigation bar TODO: Make feature work
```
Comment (gitmnk) [inline portfolio/src/main/webapp/js/script.js]: Remove TODO if already working..

File: portfolio/src/main/webapp/projects.html [hunk 11]
```diff
@@ -0,0 +1,97 @@
+
```
Comment (gitmnk) [inline portfolio/src/main/webapp/projects.html]: remove empty line

**Review by gitmnk — COMMENTED** [review]
Thanks Benjamin,
This looks great! I have a couple of styling comments.  Please fix them. And it's then ready for merge.

File: walkthroughs/week-1-web-development/examples/stanley/src/main/webapp/index.html [hunk 12]
```diff
@@ -7,7 +7,7 @@
   </head>
```
Comment (gitmnk) [inline walkthroughs/week-1-web-development/examples/stanley/src/main/webapp/index.html:7]: Is this file required ?  If not, delete it

**Review by BenMob — COMMENTED** [review]
.

File: walkthroughs/week-1-web-development/examples/stanley/src/main/webapp/index.html (hunk 12 above)
Comment (BenMob) [reply walkthroughs/week-1-web-development/examples/stanley/src/main/webapp/index.html:7]: This file was part of the demo provided to us by SPS to help us get started. It is not part of my portfolio directory. It is in the walkthroughs directory. So it is not required.

**Review by gitmnk — CHANGES_REQUESTED** [review]
Please merge to master after fixing the minor comments i had put in earlier.
